debian/Dockerfile

```diff
@@ -118,6 +119,7 @@ RUN systemctl set-default multi-user.target \
                      docker.service \
                      frr.service \
                      systemd-networkd \
+                     systemd-resolved \
```

Comment:
systemd-resolved is not enabled by default in debian

Reply:
+1